web/src/portico/landing-page.js

```diff
@@ -194,7 +195,11 @@ $(() => {
     events();
 
     if (window.location.pathname === "/team/") {
-        render_tabs();
+        const contributors = {
+            contributors: page_params.contributors,
+        };
```

Comment:
The nested structure does not match the expected type of `Contributor[]` from `render_tabs`.

Reply:
🤦Yeah you are right, will fix this soon.

Reply:
I just realized that when I will convert `landing-page.js` to TypeScript then `page_params.contrbutors` will show an error because it has no such field in it. So that would have to be fixed later.